### ISSUE TITLE
Headings are not in sequential order

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/wecount.inclusivedesign.ca#673]
## Describe the bug

Certain heading elements are not in sequential order.

## To Reproduce

1. Go to the Badges page
2. Either navigate via a Screen Reader or inspect the markup for the headings for each badge
3. See error - headings are level 4, immediately preceded by heading level 2, skipping 3.

## Expected behavior

The headings should be in sequential order, so the `h4`'s should be `h3`'s

## Additional context

This issue can be found on the following pages:
[Badges](https://wecount.inclusivedesign.ca/badges/) - all Badge titles, e.g. "The Metric Society and the Unmeasurable Learner badge"
[Events](https://wecount.inclusivedesign.ca/initiatives/) - "We Count, Future of Work & Digging DEEPer Events"

Applicable WCAG success criterion:
[1.3.1 Info and Relationships](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/content-structure-separation-programmatic.html)
[2.4.1 Bypass Blocks](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/navigation-mechanisms-skip.html)
